### PR TITLE
use a more ideomatic way of setting namespaces

### PR DESCRIPTION
**Pull request by grosser**
declaring things as cluster-service is misleading and not obvious

@zendesk/compute

### CODE REVIEW

**Comment by adammw** [comment]
Is now a good time to rethink how samson does namespaces completely rather than another stop-gap? Since if we will want to be able to do namespaces per project, this will all change...

**Review by adammw — APPROVED** [review]

**Comment by grosser** [comment]
another interesting way would be to say "if you set a namespace, we will not override it"
that will need a bit of cleanup, but could be neat

**Comment by grosser** [comment]
for now simple stop-gaps/fixes ... per-project namespaces will be complicated enough / let's not rush it

File: plugins/kubernetes/README.md [hunk 1]
```diff
@@ -192,7 +192,8 @@ to make all kubernetes deploys that do not use a `metadata.labels.team` / `spec.
 ### Using custom namespace
 
 Samson overrides each resources namespace with to the deploygroups `kubernetes_namespace`.
-To make samson not override the namespace, set `metadata.kubernetes.io/cluster-service: 'true'`
+To make samson not override the namespace, set `metadata.annotations.samson/keep_namespace: 'true'` 
+(or `metadata.kubernetes.io/cluster-service: 'true'`)
```
Comment (adammw) [inline plugins/kubernetes/README.md]: missing labels here in the path...

Comment (grosser) [reply plugins/kubernetes/README.md]: https://github.com/zendesk/samson/pull/3141

**Pull request merged**